vendor.conf

```diff
@@ -14,7 +14,7 @@ github.com/sirupsen/logrus v1.0.3
 github.com/tchap/go-patricia v2.2.6
 github.com/vdemeester/shakers 24d7f1d6a71aa5d9cbe7390e4afb66b7eef9e1b3
 golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
-golang.org/x/sys 95c6576299259db960f6c5b9b69ea52422860fce https://github.com/golang/sys
+golang.org/x/sys 9f02e9159ac345d5f0edfa7d8fc50cd5195bb2b5 https://github.com/resin-os/golang-sys
```

Comment:
This change informs the `vndr` tool where to take the contents of `vendor/golang.org/x/sys/` from. The new source implements the `SyncFileRange` system call. All the code changes and deleted or added files under the `vendor` folder are simply an updated download from the new source.